conans/client/conf/detect.py

```diff
@@ -174,7 +102,7 @@ def _get_default_compiler(output):
         return None
 
     if detected_os() == "Windows":
-        vs = _visual_compiler_last(output)
+        vs = ('Visual Studio', latest_visual_studio_version_installed(output))
```

Comment:
Given the statement in L112, this is affecting existing/expected behaviour as ``('Visual Studio', whatever)`` will always evaluate to True in that line:

```python
if detected_os() == "Windows":
    return vs or gcc or clang
```

Reply:
oops, good catch, fixed it